test/test-options.js

```diff
@@ -3,13 +3,19 @@
 const test = require('tap').test
 const gyp = require('../lib/node-gyp')
 
-test('options in environment', function (t) {
+test('options in environment', t => {
   t.plan(1)
 
   // `npm test` dumps a ton of npm_config_* variables in the environment.
   Object.keys(process.env)
-    .filter(function (key) { return /^npm_config_/.test(key) })
-    .forEach(function (key) { delete process.env[key] })
+    .filter(key => /^npm_config_/.test(key))
```

Comment:
the pattern in this project is always-parens for arrow functions, you have a few in this file that need changing